product_details/migrations/0001_initial.py

```diff
@@ -0,0 +1,24 @@
+# -*- coding: utf-8 -*-
+from __future__ import unicode_literals
+
+from django.db import models, migrations
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+    ]
+
+    operations = [
+        migrations.CreateModel(
+            name='ProductDetailsFile',
+            fields=[
+                ('name', models.CharField(max_length=100, serialize=False, primary_key=True)),
```

Comment:
I think bumping the max_length up to ~250 would give us a little more room for future API versions without exceeding limitations of some DBs for uniqueness constraints.